server/controllers/finance/reports/vouchers/receipt.handlebars

```diff
@@ -12,16 +12,19 @@
           </span><br>{{details.creditNoteVoucher.reference}}
         </div>
       {{/if}}
-  
+
       {{> enterpriseDetails }}
 
       <div class="col-xs-5 text-right">
         <h3 style="margin: 0px;">
           <div class="text-uppercase">{{translate details.text}}</div>
           <div><strong>{{details.reference}}</strong></div>
         </h3>
-        <div>{{translate "REPORT.PRODUCED_ON"}} <time datetime="{{metadata.timestamp}}">{{date metadata.timestamp}}</time></div>
-        <div>{{translate "REPORT.PRODUCED_BY"}} {{metadata.user.username}}</div>
+        <div>
+          {{translate "REPORT.PRODUCED_ON"}} <time datetime="{{metadata.timestamp}}">{{date metadata.timestamp}}</time>
+          {{translate "REPORT.BY"}} {{metadata.user.username}}
+        </div>
+
         {{#if metadata.enterprise.settings.enable_barcodes}}
           <div>{{> barcode value=details.barcode}}</div>
```

Comment:
Since for other receipt we use `<small>` could we use it also here

Reply:
@mbayopanda, good idea.  Changes landed!